lib/Resource.js

```diff
@@ -30,7 +30,7 @@ var Resource = function(options) {
   this.sequelize = options.sequelize;
   this.model = options.model;
   this.include = options.include.map(function(include) {
-    return include instanceof options.sequelize.Model ? { model: include } : include;
+    return include.prototype instanceof options.sequelize.Model ? { model: include } : include;
```

Comment:
include here is a Sequelize model class, that's why we need to check the prototype